.github/workflows/lint.yaml

```diff
@@ -0,0 +1,46 @@
+name: Lint
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
```

Comment:
nitpick: could be declare up top as I imagine that we would want to run `lint` and `validate` on the same version